clusterman/cli/util.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+import signal
+
+import colorlog
+
+logger = colorlog.getLogger(__name__)
+
+
+def timeout_wrapper(main):
+    def wrapper(args: argparse.Namespace):
+
+        # After 10s, prints a warning message if the command is running from the wrong place
+        def timeout_handler(signum, frame):
+            logger.warning(
+                'This command is taking a long time to run; are you running from the right account?\n'
+                'HINT: try ssh\'ing to adhoc-prod or another box in the right account.'
```

Comment:
nit: Since Clusterman is a open-source, we should consider using as few Yelp-specific refs as possible. Here, I think we shouldn't refer to `adhoc-prod` at all. Maybe instead specify an example ("e.g. If you're running this command for a prod cluster but you're in a dev environment, you need to SSH to a prod box").

Reply:
fixed.